projects/plugins/jetpack/extensions/blocks/map/component.js

```diff
@@ -306,20 +307,33 @@ export class Map extends Component {
 		}
 		this.initMap( mapCenter );
 	};
+
 	loadMapLibraries() {
 		const { apiKey } = this.props;
-		Promise.all( [
-			import( /* webpackChunkName: "map/mapbox-gl" */ 'mapbox-gl' ),
-			import( /* webpackChunkName: "map/mapbox-gl" */ 'mapbox-gl/dist/mapbox-gl.css' ),
-		] ).then( ( [ { default: mapboxgl } ] ) => {
-			mapboxgl.accessToken = apiKey;
-			this.setState( { mapboxgl: mapboxgl }, this.scriptsLoaded );
-		} );
+
+		const { vendorWindow } = getVendorLoadContext( this.mapRef.current );
+
```

Comment:
Key todo with this approach is working out how to load these from local resource instead of remote

Reply:
Would Jetpack's `window.Jetpack_Block_Assets_Base_Url` be helpful here?

Reply:
Thanks, that will be handy once I have worked out how to either get the webpack hashed filename, or manually set the vendor bundle hashed filename, slowly making progress on that.